libbpf-tools/biolatency.bpf.c

```diff
@@ -9,6 +9,10 @@
 
 #define MAX_ENTRIES	10240
 
+#define KERNEL_VERSION(a, b, c) (((a) << 16) + ((b) << 8) + (c))
```

Comment:
libbpf was already updated, so you should get this macro from bpf_helpers.h

Reply:
fixed